test/suites/integration/instance_tag_test.go

```diff
@@ -0,0 +1,181 @@
+/*
```

Comment:
Since we already have a `tags_test.go` file, should we just use this same file with a different `Context` block for these tests?